packages/vscode-extension/lib/network.js

```diff
@@ -53,7 +53,7 @@ const XHR_REF_TIMEOUT_MS = 3 * 60 * 1000; // 3 mins
 class FakeWeakRef {
   constructor(obj) {
     this.obj = obj;
-    this.timeout = setTimeout(() => this.obj = undefined, XHR_REF_TIMEOUT_MS)
+    this.timeout = setTimeout(() => (this.obj = undefined), XHR_REF_TIMEOUT_MS);
```

Comment:
changes here already landed AFAIK. Rebase/merge these such that this file is not included in the PR